DICUI/Utilities/Converters.cs

```diff
@@ -262,7 +262,7 @@ public static string KnownSystemAndMediaTypeToBaseCommand(KnownSystem? sys, Medi
         /// <param name="sys">KnownSystem value to check</param>
         /// <param name="type">MediaType value to check</param>
         /// <returns>List of strings representing the parameters</returns>
-        public static List<string> KnownSystemAndMediaTypeToParameters(KnownSystem? sys, MediaType? type)
+        public static List<string> KnownSystemAndMediaTypeToParameters(KnownSystem? sys, MediaType? type, bool paranoid, uint RereadAmountC2)
```

Comment:
Curious nit: Why `uint` for the reread amount? Also, might want to have a case where `-1` means remove the count entirely, since a blank count defaults to something like 4000 rereads

Reply:
Because I don't like to use special values to represent special cases, it's too much ancient C way of doing things. An amount of retries can't be negative, since it has no sense being negative so I used an uint but no problem in any case.